api/test_runner_test.go

```diff
@@ -27,11 +28,11 @@ type APITestRunner struct {
 }
 
 func NewAPITestRunner(t *testing.T) *APITestRunner {
-	run := APITestRunner{}
-	run.Node, run.NodeTeardown = newTestNode(t)
-
 	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
```

Comment:
this line had `cancel` being called after setup completion, which was giving `bus closed` errors. re-worked to close the context in the `Delete` method